Backoffice/EventSubscriber/ContentUpdateCacheSubscriber.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+namespace OpenOrchestra\Backoffice\EventSubscriber;
+
+use OpenOrchestra\ModelInterface\Model\ContentInterface;
+use OpenOrchestra\ModelInterface\Model\StatusInterface;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use OpenOrchestra\ModelInterface\ContentEvents;
+use OpenOrchestra\ModelInterface\Event\ContentEvent;
+use OpenOrchestra\DisplayBundle\Manager\CacheableManager;
+use OpenOrchestra\BaseBundle\Manager\TagManager;
+
+/**
+ * Class ContentUpdateCacheSubscriber
+ */
+class ContentUpdateCacheSubscriber implements EventSubscriberInterface
+{
+    protected $cacheableManager;
+    protected $tagManager;
+
+    /**
+     * @param CacheableManager $cacheableManager
+     * @param TagManager       $tagManager
+     */
+    public function __construct(CacheableManager $cacheableManager, TagManager $tagManager)
+    {
+        $this->cacheableManager = $cacheableManager;
+        $this->tagManager = $tagManager;
+    }
+
+    /**
+     * @param ContentEvent $event
+     */
+    public function contentChangeStatus(ContentEvent $event)
+    {
+        $content = $event->getContent();
+        $previousStatus = $event->getPreviousStatus();
+        if ($previousStatus instanceof StatusInterface) {
+            $this->invalidateTagsContent($content, $previousStatus);
+        }
+    }
+
+   /**
+     * @param ContentEvent $event
+     */
+    public function deleteContentPublished(ContentEvent $event)
```

Comment:
same problem here : the code is not deleting any content, it's invalidating cache when a content is deleted, so the name is innapropriate